index.html

```diff
@@ -2609,7 +2613,11 @@ <h3>
 <code>https://www.w3.org/ns/did/v1</code>. If more than one <a>URI</a> is
 provided, the <a>URIs</a> MUST be interpreted as an ordered set. It is
 RECOMMENDED that dereferencing each <a>URI</a> results in a document containing
-machine-readable information about the context.
+machine-readable information about the context. To enable interoperability
+across representations, <a>URIs</a> that exist in the DID specification
+registries SHOULD be assumed to refer to the information in the registries. In
+this case, if dereferencing is required during processing, the information from
+the registries SHOULD be used.
```

Comment:
I don't know what this is saying. It's vague. What does "refer to the information in the registries" mean? Definitions? Links to specifications? What does it mean to use information from the registry?

Reply:
is this a normative pointer to a non normative registry (wg note) ?... I agree this could be clearer... prefer permanent links.

Reply:
URIs here refer to contexts. There might be additional metadata about those contexts (content hashes, etc.) in the registry that should be used to ensure there is no ambiguity about which context is being referred to. Can you propose some concrete text that would help?

Reply:
I'm still at a loss as to what problem is attempting to be addressed with the text? Are you saying "The DID Spec Registries might contain information in them that are associated with URIs, such as cryptographic hashes, that further enhance the security of the system. Implementers are encouraged to use any extra information to ensure that producers and consumers are secure." Or something to that effect? I don't really like that language either because it's vague. You could also delete the sentence. :)

Reply:
Still waiting on response to this to merge.

Reply:
Dave and I talked about it... trying to propose some text.